src/plugin/electron/electronplugin.js

```diff
@@ -93,6 +103,22 @@ if (isElectron()) {
 
   // Request an updated cookie list from the main process.
   ElectronOS.updateCookies();
+
+  //
+  // Load Electron's settings file (see appsettings.js in opensphere-electron).
+  //
+  // This is currently only supported in the main window.
+  //
+  if (ElectronOS.supportsUserSettings()) {
+    // Enable mixed content so http/https settings files may be loaded.
+    ExtDomainHandler.MIXED_CONTENT_ENABLED = true;
```

Comment:
Is this a safe thing to set globally within Electron? I don't know how general mixed content behaves there vs. in the browser.

Reply:
Yes, and we do this in the default Electron-specific settings file. The problem I ran into was this needs to be set before settings are loaded in order to load settings files from an http/https URL, because the app is loaded from a `file://` URL.